analysis/combiners.py

```diff
@@ -216,31 +230,35 @@ class SumCombiner(UtilityAnalysisCombiner):
     AccumulatorType = Tuple[float, float, float, float, float]
 
     def __init__(self, params: pipeline_dp.combiners.CombinerParams):
-        self._params = params
+        self._params = copy.copy(params)
 
-    def create_accumulator(self, data: PreaggregatedData) -> AccumulatorType:
+    def create_accumulator(
```

Comment:
Maybe I am missing something but I don't see how using np.ndarrays help speed up here. We only have a single value inside the np.ndarrays, no? So how is it faster than simply doing the arithmetic with the int/float primitives?

Reply:
That makes sense!

Could you please add some documentation about this, for example to the CompoundCombiner's explanation of the dense-sparse optimization?

Reply:
Good point, done!